Library/Homebrew/cmd/update-report.rb

```diff
@@ -397,6 +397,7 @@ def install_from_api_message
   def untap_message
     return if Homebrew::EnvConfig.no_install_from_api?
     return if Homebrew::EnvConfig.developer? || ENV["HOMEBREW_DEV_CMD_RUN"]
+    return if ENV["CI"]
```

Comment:
Again, not sure it makes sense to hire this unconditionally in CI because I think it's a useful message.

Reply:
How about something that's only set on GitHub-hosted runners (`RUNNER_TOOL_CACHE`) + something for our GitHub docker image?

It's not really a useful message if it's already shipped in the images, and we are included in deliberately shipping images with them tapped.

Reply:
If it can somehow only be on GitHub Actions hosted, ephemeral runners: 👍🏻